cheroot/server.py

```diff
@@ -73,35 +74,33 @@
 import traceback as traceback_
 import logging
 import platform
+from abc import ABC, abstractmethod
```

Comment:
There's no `ABC` class in Python 2 (that we still haven't dropped), it's new in Python 3.4. Also, the canonical way is to use `ABCMeta`.

We'll be able to revise this decision once Python 2 support is dropped.
```suggestion
from abc import ABCMeta, abstractmethod
```